src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -664,7 +664,8 @@ export default class OCPPUtils {
         const meterValueWh = meterValue.attribute.unit === OCPPUnitOfMeasure.KILO_WATT_HOUR ?
           Utils.createDecimal(Utils.convertToFloat(meterValue.value)).mul(1000).toNumber() : Utils.convertToFloat(meterValue.value);
         // Check if valid Consumption
-        if (await OCPPUtils.isValidConsumption(meterValueWh, lastConsumption.value, durationSecs, chargingStation, transaction, tenant.id, meterValue)) {
+        if (meterValueWh > lastConsumption.value
```

Comment:
Why do we keep this check in the initial if statement? We will not be able to log this error later in this case right?